Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -579,7 +579,7 @@
 
 void RemoteLayerTreeDrawingAreaProxy::hideContentUntilPendingUpdate()
 {
-    m_replyForUnhidingContent = sendWithAsyncReply(Messages::DrawingArea::DispatchAfterEnsuringDrawing(), [] { });
+    m_replyForUnhidingContent = m_webProcessProxy->sendWithAsyncReply(Messages::DrawingArea::DispatchAfterEnsuringDrawing(), [] { }, messageSenderDestinationID(), { }, WebProcessProxy::ShouldStartProcessThrottlerActivity::No);
```

Comment:
Curious what @mattwoodrow had in mind here. I don't see the point of using a sendWithAsyncReply() (instead of a simple `send()`) if we don't use the completion handler AND if we don't take a process throttler activity.

It seems the only point must have been to take the process throttler activity.

Reply:
Per offline discussion, we'll probably have to wait until Matt gets back from vacation to understand the original intent here. In the meantime to work around the regression we'll put a timed activity in place.